frontend/Views/Widgets/ViewNemesisStats.svelte

```diff
@@ -0,0 +1,312 @@
+                                const value = context.parsed.y;
+                                return `${label}: ${value}`;
+                            }
+                        }
+                    }
+                },
+                interaction: {
+                    mode: 'index',
+                    intersect: false,
+                    axis: 'x'
+                },
+                scales: {
+                    x: {
+                        stacked: true,
+                    },
+                    y: {
+                        stacked: true,
+                        beginAtZero: true,
+                    },
+                },
+            },
+        });
+    }
+
+    function applyFilter(filter) {
+        selectedFilter = filter;
+        filteredData = filterData(allNemesisData);
+        updateCharts(filteredData);
+    }
+
+    onMount(async () => {
+        loading = true;
+        const data = await fetchData();
+        if (!errorMsg) {
+            extractVersionsAndReleases(data);
+            filteredData = filterData(data);
+            updateCharts(filteredData);
+        }
+        loading = false;
+    });
+</script>
+
+<div class="container-fluid p-3">
+    {#if errorMsg}
+        <div class="alert alert-danger" role="alert">
+            {errorMsg}
+        </div>
+    {/if}
+    {#if loading}
+        <div class="text-center my-3">
+            <span>Loading...</span>
+        </div>
+    {/if}
+    <div class="row mb-4">
+        <div class="col">
+            <div class="card">
+                <div class="card-header">
+                    <h5 class="card-title mb-0">Nemesis Statistics</h5>
+                </div>
+                <div class="card-body">
+                    <div class="btn-group mb-3" role="group">
+                        <button
+                            class="btn btn-outline-secondary {selectedFilter === '' ? 'active' : ''}"
+                            on:click={() => applyFilter("")}>All</button
+                        >
+                        {#each releases as release}
+                            <button
+                                class="btn btn-outline-secondary {selectedFilter === release ? 'active' : ''}"
+                                on:click={() => applyFilter(release)}>{release}</button
+                            >
+                        {/each}
+                    </div>
+                    <div class="btn-group d-block mb-3" role="group">
+                        {#each versions as version}
+                            <button
+                                class="btn btn-outline-primary btn-sm {selectedFilter === version ? 'active' : ''}"
+                                on:click={() => applyFilter(version)}>{version}</button
+                            >
+                        {/each}
+                    </div>
+
+                    <div class="row">
+                        <div class="col-md-3">
```

Comment:
Re: Breakpoints - Check here as well, there are also `xs` and `lg` breakpoints where these sizes might be problematic

Reply:
It was looking quite well - try on staging